LykkeApi2/Controllers/OrdersController.cs

```diff
@@ -273,6 +282,100 @@ public async Task<IActionResult> PlaceLimitOrder([FromBody] LimitOrderRequest re
                 throw;
             }
 
+            return Created(Url.Action("Get", "Operations", new { id }), id);                  
+        }
+        
+        [HttpPost("stoplimit")]
+        [SwaggerOperation("PlaceStopLimitOrder")]
+        [ProducesResponseType(typeof(string), (int) HttpStatusCode.OK)]
+        [ProducesResponseType(typeof(void), (int) HttpStatusCode.BadRequest)]
+        [ProducesResponseType(typeof(void), (int) HttpStatusCode.NotFound)]
+        public async Task<IActionResult> PlaceStopLimitOrder([FromBody] StopLimitOrderRequest request)
+        {
+            var id = Guid.NewGuid();
+
+            var pair = await _assetsServiceWithCache.TryGetAssetPairAsync(request.AssetPairId);
+
+            if (pair == null)
+                return NotFound($"Asset pair '{request.AssetPairId}' not found.");
+
+            if (pair.IsDisabled)
+                return BadRequest($"Asset pair '{request.AssetPairId}' disabled.");
+            
+            if (request.LowerLimitPrice.HasValue && request.LowerLimitPrice.Value <= 0 )
```

Comment:
Put all validation logic into the model using IValidatableObject
https://stackoverflow.com/a/3401178/8592182